src/styles/VersionHistoryPanel.module.scss

```
@@ -0,0 +1,179 @@
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    text-align: center;
+  }
+
+  .diffContainer {
+    flex: 1;
+    position: relative;
+    overflow: hidden;
+
+    :global(.ace-diff-container) {
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)